commands/buy.js

```diff
@@ -31,7 +31,11 @@ module.exports = function container (get, set, clear) {
         var order_types = ['maker', 'taker']
         if (!so.order_type in order_types || !so.order_type) {
           so.order_type = 'maker'
-        }
+        } else {
+	  so.order_type = so.taker ? 'taker' : 'maker'
+	  so.order_type === 'taker' ? delete so.taker : delete so.maker
+	}
```

Comment:
Hm.. Have tabstops set to 2 without filling with spaces. Expands in other environments